packages/apollo/test/components/Mutation.test.tsx

```diff
@@ -37,7 +37,11 @@ describe('Mutation', () => {
       },
       result: {
         data: {
-          updateSomething: {},
+          updateSomething: {
```

Comment:
FYI test was failing because this didn't return data.  No idea why it failed below and not here, but it was a promise failing so maybe just jest catching it in the wrong place?

Reply:
Thanks for fixing 🙏